packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -69,7 +70,7 @@ export class DataBuilder {
     const { organisationUnitCode, startDate, endDate, trackedEntityInstance, eventId } = this.query;
     const eventsProgramCode = overridenProgramCode || programCode;
 
-    return this.aggregator.fetchEvents(eventsProgramCode, {
+    const rawEvents = await this.aggregator.fetchEvents(eventsProgramCode, {
```

Comment:
No need to distinguish between "raw" and non-raw events in this function now

```suggestion
    const events = await this.aggregator.fetchEvents(eventsProgramCode, {
```